css/properties.json

```diff
@@ -3240,6 +3240,26 @@
     ],
     "status": "standard"
   },
+  "font-variation-settings": {
+    "syntax": "normal | [ &lt;string&gt; &lt;number&gt; ] #",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Fonts"
+    ],
+    "initial": "<code>normal</code>",
+    "appliesto": "allElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
```

Comment:
Please use `perGrammar` here. (It's actually the same, though it's what the spec. says.)

Sebastian